packages/data/src/settings/selectors.js

```diff
@@ -50,7 +50,7 @@ export const getSettingsForGroup = ( state, group, keys ) => {
 };
 
 export const getIsPersisting = ( state, group ) => {
-	return state[ group ] && Boolean( state[ group ].isPersisting );
+	return state[ group ] && Boolean( state[ group ].persisting );
```

Comment:
I realize that `persisting` is the property on the state. But since this is a boolean value shouldn't we just rename the property to `isPersisting` in the state (and make the necessary adjustments in the reducer as well). Convention is to have boolean properties represented by the `is` or `has` prefixes.

Reply:
persiting -> isPersisting in https://github.com/woocommerce/woocommerce-admin/pull/3430/commits/aaea4456c7c7000fca09f1bd0f3215666c57f2e4